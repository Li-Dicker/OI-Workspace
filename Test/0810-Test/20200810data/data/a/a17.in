10
196
11706 79448 87309 77651 90469 54318 85533 62745 45391 76790 7322 9464 42691 47344 20373 53340 30195 63697 20191 82349 22445 99338 59871 57239 45771 95058 70967 43889 76139 57466 18328 58715 21838 77974 45072 30180 35054 46273 49093 85151 38893 74142 5148 63199 84990 94440 31025 69919 11348 77943 212 66672 9860 14509 50715 30435 38604 44851 2540 25652 87393 1952 30161 14 63851 66179 41153 31759 34394 39254 15692 36976 61322 9267 81567 20075 73025 3387 82082 25399 48768 54373 83631 32039 69363 32066 13959 11972 27760 59270 17436 65810 3047 69054 3193 51308 77614 37272 62344 35763 78093 89239 78993 94855 69641 59705 76629 79984 79065 50652 61650 47992 95575 25480 1580 71106 93710 17862 58202 88359 50691 96321 6772 43042 70800 8966 25307 50683 89964 4217 80730 3535 96509 35809 88554 89639 43962 27818 46763 33182 59000 94610 38099 54572 81746 12405 32644 58458 60947 80169 65529 59136 56179 30421 64576 90252 85807 26089 41966 87066 37390 98050 79726 55685 39738 66029 39717 27366 10161 91147 37062 15707 47271 11725 61695 75032 55342 52701 37347 50622 58774 88201 15147 12494 19500 83450 73529 79380 62996 68939 11644 69137 77834 56062 34794 58615 
4243 19213 12773 19634 44676 47611 12915 43244 7367 9369 429 3703 49592 31404 870 8909 40875 8073 43931 27728 1290 29514 3209 32048 24977 47615 44782 41043 41231 47677 37351 18691 24494 35427 6992 32902 14461 35686 38377 44426 15718 46119 30204 16633 32329 14652 10148 2383 17131 14553 20372 32400 13532 23908 17620 17000 39364 21749 22180 34086 14115 46847 12449 24721 35932 11426 49744 10707 34383 6128 151 9520 4619 19290 28645 28778 10937 524 45695 12180 15350 13423 26645 1636 22549 39114 12415 7592 34383 6291 9554 43954 27098 6416 27194 197 24608 2733 32941 24487 39450 30678 42232 34062 27017 7536 34216 23603 24905 43189 43232 20748 5047 48485 22594 48219 7860 21034 13632 20402 1260 49191 17781 34789 18666 40556 47655 5132 15547 47664 29544 23882 2276 31096 32048 32209 31019 19136 41260 1832 995 45094 3989 29492 4342 27680 14643 28718 21013 26594 25283 37519 45036 1820 42228 25246 49327 49617 5740 36810 32384 34505 6304 2043 48572 21189 5658 47794 639 24809 655 1748 11465 8986 18507 19443 20332 13461 17453 13848 11875 4759 6171 25519 3815 9064 46892 33952 10684 35349 5102 31774 20494 18804 40996 32081 
199
67541 72119 28021 82192 30301 12649 9949 62507 21146 98319 85960 215 62510 37361 42811 17688 44346 98752 51662 83566 6499 73802 51481 61651 94732 36714 56825 3454 15174 63999 40289 65758 36361 99872 15700 1212 7292 99766 74063 35446 82877 58595 60730 59100 46747 55737 50640 71917 2074 88113 89079 63703 74843 61171 28117 49730 88479 92366 22600 47368 92959 79211 15934 40650 56059 79910 85475 94476 10743 91250 91964 40880 98260 61765 69228 15649 9478 91359 54097 79748 67272 6338 13704 6076 93654 6780 80233 19201 92837 84232 62035 97065 8060 44456 45394 38163 81740 66112 49989 47126 27214 57738 42202 79019 59283 36412 28180 28022 10276 18943 6422 45488 99877 48269 8554 78002 59368 49938 29018 50761 40756 47413 96689 89492 28247 80477 52280 72481 7566 30442 2134 98532 90745 89621 55863 76583 33120 50147 63983 1856 88958 64067 37741 31027 8971 54543 76396 87517 46988 95970 41466 74250 47116 97222 30049 71250 68750 55491 78274 80920 90343 88936 82266 99374 25720 90287 8117 69911 10061 31554 27826 65904 34 37591 30700 18022 21728 25058 34692 96879 49574 16951 74914 70366 5341 29235 21521 41461 26740 83500 14217 49956 93128 49701 52186 56250 93889 82909 62817 
27448 36710 16082 35764 33168 28445 46622 26462 15861 19847 42917 22782 19982 26907 42679 21046 16563 38894 18046 14732 28552 30505 14420 24971 4195 10135 687 14579 16869 35854 18383 31720 9454 44339 29212 38750 35570 18922 26517 20689 9455 16437 25291 11981 27551 42242 46431 38147 19412 43477 31147 40403 18281 45605 34937 38690 26819 43530 37115 8081 26234 46764 17969 38301 33132 7127 17523 4640 4852 21078 35484 9197 40736 48576 28281 31103 40233 44616 25532 48101 26305 28799 8565 9191 26233 1744 37769 5545 32254 19078 25346 30331 40341 8217 33388 20014 29100 22756 26162 9036 8874 49507 32766 37618 1011 25997 31137 46086 41110 46562 16821 4107 9258 2537 4357 513 16476 19028 40791 1844 40458 31680 37728 46265 24315 21029 35269 1108 21770 26185 5394 43098 20332 9476 36937 9486 12634 41233 32065 47730 46284 9222 29206 27995 45080 47563 17786 31390 4474 39307 44620 34936 1343 44915 20044 27358 33959 1851 42520 44807 44880 41627 31713 23124 15016 43532 44824 47017 12097 19556 47899 23741 2951 24782 38806 29199 21319 7436 48091 32933 41878 29569 17131 43546 5216 42681 47760 12015 3328 28033 22655 25356 5608 47859 14685 7509 7690 41408 18765 
199
67697 24247 244 72613 54122 72143 48117 40065 10494 44516 55239 6080 9921 45799 38836 33096 15731 20498 88833 27865 59621 23335 9736 58022 38 61194 6369 81574 72395 86633 22167 40589 24579 37926 44901 9723 49303 78584 84246 25076 43734 85096 78978 3123 97988 46716 92801 91927 25107 11017 24734 89113 30048 80071 6362 18905 37716 98642 73406 61638 71861 70384 14816 16320 40909 78563 66008 23189 3144 1987 69527 39524 3627 62508 64664 47715 85384 6250 30305 55202 95024 85295 20557 24211 38240 67373 67843 78218 95311 39967 45264 96976 14209 69369 31647 44648 35981 14694 83764 30171 30476 58964 88605 45247 29943 67461 54664 43027 72964 87513 90933 38016 49590 21742 39142 92965 41671 73900 21466 73227 88003 16825 43102 66185 5126 54894 23800 45485 45466 67381 75946 93597 93603 11902 9749 17111 78840 5357 43870 77535 15798 43978 61746 34032 32674 49617 68162 68178 9161 40832 7553 21443 47379 13426 37222 65937 71718 54163 84854 60212 85279 39623 33346 33987 3976 5120 84837 18654 26230 66890 12476 21638 75217 73221 49559 94492 71750 78419 37674 40742 62038 86235 86176 58455 52491 95889 79870 76512 54119 78008 62586 74278 94468 14653 94236 48734 91134 1008 11527 
13276 2895 11598 6738 49146 49155 28375 35591 40411 23954 44601 9893 12324 3504 46155 43160 31428 10035 41486 36277 20491 41492 31982 28333 28489 32665 8902 42468 35520 25150 26183 39377 7665 25724 2887 9089 4577 27102 32592 26539 42693 23758 27182 21235 31800 47206 10031 6940 31243 41877 32544 32835 48010 27955 14631 14494 38002 21717 4298 17101 35514 7003 45736 39962 38399 3131 8249 16501 2583 6042 11462 27480 35257 11405 26814 39275 1740 41200 39287 24248 33210 47270 20517 15311 7678 44735 5103 12880 46328 11753 22012 40691 35556 37807 8029 42172 44637 32028 2333 35538 49934 23937 2760 42543 34829 37956 25413 12308 8229 43462 41379 11290 20950 8286 49405 47201 27314 22886 13639 31434 5404 43812 11889 20161 36542 40345 29102 16028 29349 48300 45371 49322 42650 29772 17371 41085 34744 1377 19036 32423 39023 7124 8203 48091 40529 47396 3877 6088 34984 29325 36286 35724 34379 47904 37917 12413 9692 42197 177 17416 28578 40926 35476 4787 49921 33554 38253 27234 39308 21662 19250 43101 44321 38746 14453 11578 45918 43552 7584 22703 11623 40290 13212 22643 32071 21302 3058 16438 48302 31683 46646 11544 29761 16141 28343 2288 33304 29455 13874 
197
22451 79042 32779 61549 12598 34380 45571 71344 31333 38162 20820 88762 74808 90591 18585 89389 56611 26718 3189 25675 30803 7636 77955 2544 50064 81228 44201 71266 39821 72329 16405 49422 62909 46666 10432 83475 186 92327 6376 80268 99063 72606 48335 20690 48819 24102 46567 18654 11281 66446 13138 13540 73273 49791 20237 71840 14880 10551 59922 63450 4529 53119 51827 25228 8238 51629 52306 16117 63487 74321 87925 26699 29418 24591 22446 71906 62161 71840 57277 8791 12769 69653 32714 75228 10692 96535 84512 10845 1013 18390 84391 72843 10100 25666 89652 30608 10284 17542 77643 60381 8913 76629 29579 13591 43862 20145 2623 11264 87518 29110 59715 35749 84954 66464 51515 23198 16453 48953 83875 65158 38119 38949 11302 12625 35704 32982 14306 48022 71714 86088 56527 774 52517 8519 83722 91358 16407 41194 90776 60142 50096 48786 13719 85142 37015 10120 77226 14013 23237 90046 13351 26155 6711 29584 91947 81301 18119 60483 56000 7412 6527 79194 39289 2057 64564 95651 24166 30182 99602 79301 44692 89076 22566 88349 57695 32917 63444 99660 7880 20627 38898 89811 49908 6667 72592 24502 54485 19757 2067 49481 841 24008 17410 40225 94334 15972 33397 
5678 31619 23190 142 12365 17103 4443 26830 38581 25055 14690 19767 31338 11006 32748 37295 25703 35455 24206 20913 21909 45803 8215 23807 14889 25212 34613 47269 24157 19330 23403 18444 22292 34144 35032 8578 47199 8920 5744 7247 12019 30571 39141 2022 39105 32118 39212 22331 44044 47095 42956 49464 25896 5588 35748 3584 9426 3451 16049 7780 7425 5557 41118 20719 30588 17123 28307 1628 9075 9766 40750 11265 27597 35818 16218 44493 10742 15845 9406 38459 16365 38457 6284 49683 45080 12638 40495 29844 6663 21045 33286 43499 18930 29872 29868 42484 33085 34964 44479 38925 39477 8167 28884 26403 49891 25913 27443 3333 44972 22122 48281 30767 25865 14643 34633 42899 9219 38943 30159 6663 40856 34153 45405 36389 8419 9140 34579 49876 43327 2768 20549 37956 6439 23746 16131 38784 31085 21831 9458 34673 48472 12277 39037 41253 2317 32155 7768 5228 33934 6518 32769 28468 41417 20961 17696 49512 45688 46365 25036 1122 16950 2894 49402 49003 10746 12260 23612 39043 29571 13500 45484 24316 15478 38643 28693 17794 20987 46093 19599 7999 14518 32871 10843 28985 9834 37631 40592 27299 22550 25645 5466 38895 20617 49077 14920 31673 9437 
196
3133 76160 93994 11334 53901 6456 66424 65998 75307 12750 79219 81766 76832 69624 26747 8644 97106 20478 94824 12494 18817 43100 78462 78956 822 57528 95487 31129 94542 34225 30069 24129 8562 1754 73105 83447 34914 12760 94886 52306 4289 10195 93673 53390 48228 44399 21157 81236 7815 87027 66915 56296 96589 72223 11345 59947 46835 66688 41950 93384 68885 15700 7672 86360 46807 45693 97161 98726 76490 68086 13077 2930 28315 81667 56848 34918 62381 45059 14585 38590 78383 2499 44932 36693 13708 16540 2598 80077 87336 12134 11725 18377 43928 46942 59802 36493 21829 58525 44759 64521 86080 23589 42609 86308 30617 37708 35051 50829 90117 43235 14667 40244 78065 63469 50134 3700 49679 958 80333 47064 80983 76479 43182 74931 23025 4415 69507 4419 12763 77800 9193 40724 58123 43443 90397 85586 84791 65752 74831 86545 3461 18514 45416 53674 66481 45070 89053 25403 8816 60000 30254 57312 38960 89780 94429 17138 94836 82644 41284 3653 84050 21298 6889 34703 70 6512 14180 39812 39619 41895 47553 99313 2649 8077 85077 72866 50717 42194 77889 48434 39535 82410 11557 51308 23306 34649 41646 20562 540 23626 90968 67449 19880 26555 65604 71881 
2729 21659 43273 41185 3443 6954 24408 25862 16470 25518 20830 41143 41519 1411 38707 4806 16175 2251 23337 19881 3295 10204 32697 22624 37522 41043 14319 36343 42970 7626 34402 26638 36679 30635 849 21460 5206 9305 37171 17269 49197 44904 797 21613 23364 14235 43080 17918 25580 29698 7674 49687 29899 21373 48788 41944 12275 7398 12319 39525 20123 35787 24494 4385 49336 15451 44658 21822 3262 27320 1211 18820 36196 41946 6103 8559 36558 3693 37307 35995 12060 15951 36438 26499 4608 40128 9170 13493 41747 35059 13162 17154 34213 30068 16761 37469 44704 41654 35936 37844 39854 33458 6706 23652 7671 39787 3836 36170 16774 20141 32945 39091 27201 43066 41914 27938 27967 20973 49904 48340 24066 38510 33262 12328 979 36708 42110 28685 31578 43742 14180 34005 24153 34619 12458 27995 46916 20842 33155 27889 18033 35678 30531 34099 16378 19705 19107 39528 7056 29439 38417 7525 1158 15030 1425 20954 30235 547 3469 40440 2914 27959 40629 15177 6004 8078 22481 31169 46551 5135 43914 10301 49146 8260 974 38655 20457 4508 5120 38887 30000 35564 48353 19349 14921 33922 30952 28198 9768 5408 44495 8298 18928 5340 42359 657 
200
93909 75124 65266 67448 72914 26342 50235 7850 80060 75116 95041 99416 28248 52970 56613 86444 13230 32337 32104 69301 53545 71740 30230 2447 28533 35637 76902 65557 78509 51190 17414 68062 23234 96790 79888 64578 45951 30857 8344 59226 13327 65922 1530 6587 54162 98874 90337 62081 76687 27674 20980 34683 84876 19204 37220 1916 55967 8163 66680 62311 78461 78142 57528 93034 37856 14027 695 8907 82100 44294 62941 28164 35754 32676 34142 48126 66904 23564 37448 66143 25411 38541 97303 55992 12241 40076 4070 49053 48553 67313 34805 81058 72504 63412 76452 29440 43495 72533 58165 53058 58043 30790 79335 82349 65597 61304 66974 34470 80023 57434 34570 68730 76681 53279 99746 52822 54968 68828 43794 19734 55392 62623 63571 45509 49599 85095 24062 86631 65626 70524 4224 86993 96246 69162 83777 31867 65194 12915 23268 2584 95752 21223 39777 87389 31400 7692 22880 93945 52156 97549 84100 66695 88225 40752 79356 30198 47883 99854 32167 59279 47797 46941 74408 91043 14596 54618 7111 11617 42685 71754 35918 99055 24260 81472 60288 90524 74081 88441 44903 2334 86985 62394 39679 64114 96138 21847 31671 91603 58015 51319 50628 56752 82715 98323 89558 96377 66646 71033 4997 33385 
38911 16599 11623 40641 25267 16802 26188 6386 17167 28187 20825 13322 25995 27066 26802 26492 33417 31946 12677 21676 26763 35770 36259 21819 24070 17217 29241 28998 24323 23698 25078 13195 37970 20119 45689 15365 28546 34658 2066 34070 18780 3468 11714 28954 24171 45714 12273 8413 46764 19959 45285 40232 22172 49626 22495 25545 49120 22814 43031 43276 37649 7348 3985 15722 12338 23393 28157 39623 37467 8920 45675 2111 47458 13932 6622 16159 23018 8738 38528 4277 2036 34781 17893 30347 18782 22101 15632 30626 40946 40633 19351 29092 7882 7591 46994 29057 22839 34986 7397 4038 36275 49837 16062 40075 5727 25713 37745 36973 36069 25747 4954 5753 34168 42188 41986 11627 20139 18625 47184 45603 15518 14037 5208 36760 675 1965 7301 34516 14732 47953 3523 35101 1259 3234 37836 43083 44724 26690 29649 14404 14139 29789 10635 29322 17399 18698 13539 40907 20511 15702 8269 25595 24671 9887 33802 18267 24052 8681 39506 27557 12955 14625 5275 3086 10155 22847 26280 21766 28024 232 26949 15302 36298 7003 14309 8719 14945 17669 26805 31558 26560 49110 2449 24330 34373 29480 5253 14575 49848 36622 37506 17780 48653 104 42398 25182 23716 4780 29420 44665 
197
76562 33455 28377 88729 50828 48339 15709 44065 44476 85264 63434 59574 56969 13509 13396 78914 22918 33944 83779 3942 95366 13626 51365 49668 38826 21622 82947 85251 49440 76829 96366 23176 72071 98745 81560 50833 56396 15563 55587 86705 30129 2135 90189 80607 89352 92896 25350 26019 86651 57412 54688 84429 16154 98859 80002 22715 18319 20907 91348 4499 83243 82300 17242 2913 79563 37509 96069 29577 36586 94448 67260 29524 27604 96076 2194 54506 81217 56901 19485 14801 8970 18344 65307 87682 59316 95112 98987 55511 19004 35351 22771 94670 81986 14763 39463 30287 47184 30987 78551 93851 72908 71419 82299 51658 41667 67117 92108 67275 63969 10760 49559 86091 80669 30526 25404 59716 75029 60861 20381 7677 84931 71503 9157 51654 49121 22099 54372 57684 8371 65014 50333 66761 85925 49294 12521 84779 24563 56032 68977 31422 14776 44982 18458 23118 77843 98088 67881 66285 98797 10482 95284 19064 43756 81690 519 97841 75075 1328 73052 71473 47418 50219 83600 82766 89915 11121 60111 11397 44915 28404 23779 39708 66740 97570 11026 28871 24053 67580 69739 17255 48031 35220 65144 45933 6762 69084 28795 94659 96068 98636 55775 55127 27167 11957 60169 68333 74356 
41988 29409 20167 42132 46941 11348 29797 19777 23120 49445 20458 190 4638 22853 38532 21115 49866 13514 23861 32387 1799 25104 26260 45272 40893 26914 6637 12856 5422 48710 38051 22524 26361 31438 6489 39082 45377 15237 34625 18615 5166 48773 39195 30921 40249 1025 38877 18864 3880 26801 30968 45023 25310 44422 13795 49238 43672 19163 2281 12856 20757 42304 774 40440 46179 30134 41305 46813 44101 38774 7163 22455 14169 13932 8024 33716 6125 46792 25929 2744 17390 35739 11839 37241 26627 27186 14989 43565 10581 41024 48831 4071 11766 13507 1344 46065 44392 49663 24818 21192 33649 19223 6206 10981 6857 17335 15767 16892 12770 147 36098 36573 24739 32424 39426 40439 35936 19292 12800 46218 45928 13907 39201 25499 19020 23363 16151 35259 42202 20399 15146 17781 23185 57 27828 24903 35168 10737 37436 34260 26612 2531 26121 12091 35411 49625 13237 8090 22727 29025 5697 29951 4045 32664 5332 14409 38206 17402 21278 11508 4326 47857 75 4866 24965 15424 13588 20440 35479 14851 14967 16184 31620 48754 27595 5530 25819 21833 3108 34012 23420 35045 23521 12730 7894 45237 26025 19101 31940 9783 22764 12743 48855 45178 37721 22476 30921 
200
43213 20129 9387 8683 8372 66948 39048 41821 35722 84900 21925 11437 13733 75239 4555 10632 97492 91482 86956 30431 65791 63448 88080 24416 56317 32830 27110 78407 79604 82060 43333 83297 80949 19298 83974 40642 91998 18848 94420 86573 51191 4216 17620 64369 85458 102 56345 25480 27072 97612 18635 8619 5213 71572 55471 91865 76072 88375 38738 71492 52015 12216 51690 49999 3727 2581 4500 34935 99252 69862 95072 38496 58920 20584 72301 65984 84029 7709 37876 38766 64509 64753 48797 56489 51977 88656 96283 87031 67166 87148 49247 52062 24020 868 5281 39415 99100 27869 80149 41439 8412 47160 34150 44460 52253 27076 99246 68577 26271 22574 11940 72052 77838 55613 27587 28736 28703 82837 46319 4485 34500 28722 9503 18417 86194 89163 75094 86361 43295 29920 3711 55816 76312 90914 26262 40001 13199 33145 29810 75268 7579 95720 98927 99437 84252 18205 55994 98129 417 40307 26688 51815 61541 99660 95203 45746 42107 26182 86336 66374 28937 37220 49922 31305 50143 59380 95655 44278 637 2459 41129 38352 12011 18891 61627 8015 50107 54746 85860 95018 88046 60287 73210 89143 54887 35677 26125 30668 51249 53076 46140 81504 28880 53440 14950 30800 19517 97788 55241 91130 
6291 18367 31005 25299 31891 32346 45750 33515 32324 34421 27702 42587 16092 11695 48761 15778 35133 48929 23527 15491 17107 48732 7056 47985 15880 26999 30932 40109 40831 24787 24904 8849 32102 28237 29097 26012 26444 8769 17770 27410 39443 19154 7405 6913 18175 14100 6887 15179 27251 23879 27479 13798 44564 22174 30528 42289 38447 27261 5657 33886 25728 45763 36497 10955 39629 2436 48451 41317 48191 16304 26218 31013 29667 27732 38080 21978 34848 2126 14789 48195 24156 3225 7314 38653 23670 28506 45043 23061 26473 739 17030 19686 40026 699 22485 19998 13651 41085 13484 28951 497 39213 36196 26854 1529 13027 40718 40341 25021 4860 29276 32879 8206 39478 30548 42255 41257 4383 41235 26645 39911 39958 14971 4164 20192 21079 31074 245 21691 17740 340 9321 27125 16547 15570 13304 146 31003 28353 28982 29305 34461 39621 37592 5552 27243 16085 30225 23457 37858 41113 5226 45247 28440 36677 28463 19954 10455 41514 33171 21945 5934 671 7399 31436 49804 41861 10475 30283 1766 40908 18694 14541 40363 13683 736 12669 45566 34559 32996 31936 23518 46312 45430 33782 5638 24466 41306 2362 3427 47196 26804 2283 2694 15456 49828 29240 26335 11214 7109 
198
34412 61973 89479 30308 81552 98879 22318 66624 80450 22699 36341 68544 23514 27979 8522 19687 50648 33271 21929 67097 2745 65950 97398 91467 52816 211 63619 81886 33241 36586 69257 40304 99396 87662 77120 93856 68477 5280 4763 78123 1441 57617 13310 47535 38229 83108 90729 83682 89578 29450 88360 10551 52702 60326 10710 64746 39062 94136 60139 27360 3249 81085 64273 5777 10035 40973 99498 75408 71501 18103 62682 23399 87757 6288 28877 79320 62893 96057 34649 60316 84996 44734 85158 75504 45205 12544 1072 23917 41100 87150 68349 22989 88799 68202 36203 95411 5109 53068 70027 757 69037 31275 50803 61623 62503 28788 49715 11861 94714 86303 23939 91407 44356 56307 1147 95956 68530 34848 31426 31287 57467 13072 63116 59382 98668 23914 71840 23054 34716 19431 84048 81001 52513 32452 81470 59916 65181 99884 26103 45662 52347 27527 82981 67931 93183 26643 8782 25634 25678 81885 22294 34452 32478 92681 73511 43537 54637 62867 74355 91260 65343 74603 23890 41932 67577 17258 40500 6839 79390 45199 95310 59754 34700 35829 23138 31703 40608 37987 95207 31313 32651 26371 3836 28485 62991 56417 38257 97149 63806 9353 63310 15573 79871 40093 52128 63562 76249 69973 
37179 8381 24546 49274 23635 4357 31314 32799 45163 32104 43741 27187 19922 17441 47043 35961 5786 34423 3654 2575 33151 36753 43167 9526 17060 46100 40735 31403 47230 36296 38782 27376 11919 29893 2927 13319 44836 29993 5144 28784 23448 48851 13438 44188 49406 9660 13282 3701 48917 33180 9039 28370 30728 12541 44682 9919 36112 14406 30738 25349 7979 1685 8161 2067 46332 40211 41737 41107 28731 9400 23697 1610 301 16472 39824 21908 29643 18245 47226 21650 4394 24189 22540 27028 23678 46599 21701 23885 14714 15666 35124 20887 4234 1035 15684 21979 9546 33142 28083 11263 27595 12235 27852 938 12838 48793 14484 30708 42221 29749 6651 6031 32744 13004 4008 16875 32206 43741 3385 34547 23727 6798 4058 39074 2747 34014 7409 29200 23504 24570 44563 41393 1609 7847 17817 26121 48479 12725 35167 17506 6287 2136 17963 37212 372 37923 49160 30329 4769 44365 15803 40536 32337 598 18803 32340 14060 39262 15252 1675 22871 31771 39284 46008 40630 31426 39287 31302 11572 6559 24527 38545 23778 46796 21706 31417 572 14129 22658 4643 49157 1589 38116 40510 30385 18100 17105 29673 42504 41282 30455 31494 23356 42343 33843 8245 25186 4020 
197
12356 438 20007 78743 65754 97498 17315 98520 87118 17419 57544 42448 78879 19297 89890 51214 31421 10513 61852 60955 29488 66092 34435 91476 93804 92470 32606 57050 12603 44909 22411 8132 35024 14638 36847 89882 58767 8219 40234 98792 36763 46229 89947 35927 56160 97099 28302 50789 2819 32653 41617 35509 33297 76890 51845 4586 98874 47062 8201 30114 17301 55966 31532 70864 25979 9156 45230 38345 51040 58307 24425 11483 88022 84710 41841 76364 49734 33106 85665 39014 91268 53022 42980 48990 21459 58387 44388 25406 56367 80428 69856 75238 58202 4393 75516 67044 57439 25591 40448 31450 13478 11163 68926 98486 46615 9629 15996 44085 34605 62037 92825 78499 41032 55862 3892 51374 7522 44355 19282 20639 18030 80907 30214 43101 52953 62509 98445 36391 78506 21918 70332 91211 36739 74290 20236 250 74163 19204 67701 4712 71064 23198 77728 51620 52299 36734 47596 7320 21427 16026 67371 11212 12472 17089 13238 5855 68359 53468 26345 72856 32006 67444 33549 62769 96785 42159 82454 9754 78067 88523 60215 99065 71939 85771 3988 91597 36565 12663 75898 68489 12869 12214 61422 94934 62033 8384 95445 15863 27127 82358 73912 11098 3171 57969 45164 4285 44752 
18062 16701 18215 23991 38494 27937 37702 10815 42883 25274 24452 3203 25394 24774 39157 37650 31022 4344 21879 37728 36884 43221 41658 12598 15494 49753 24212 26881 46413 45127 17446 24646 44702 37316 27778 36198 26293 2375 13485 35530 23077 28075 18392 19853 44477 11704 48619 18079 18351 21353 33829 47243 34337 2628 43968 4786 1587 45117 10264 16376 30116 39844 3950 5746 26771 19097 47245 10113 1061 29453 33178 11765 5317 16114 12396 37558 10431 21040 32666 49881 29194 83 42180 7145 49251 28338 26368 40954 35744 5604 35649 26632 25304 4774 13627 32056 17308 40747 36550 8617 26539 37553 26716 17895 48514 9851 11223 6086 11471 16686 8459 9369 14166 47887 31029 40312 33645 2545 16709 37755 4105 33720 42034 9814 36753 21752 39498 13353 11441 16547 49421 36633 39908 8148 39880 30571 9850 1437 29288 45867 7886 32873 22835 43183 28725 6603 37567 2692 36399 45003 44234 8420 29000 39595 33484 29164 34746 49170 29095 42489 5975 25199 45791 16341 28325 25853 43306 21652 45123 26866 33765 30136 25473 49039 31956 43172 23488 817 8589 29582 9531 2020 30275 33720 15219 47453 6151 38547 41294 3327 8208 26125 25690 15449 24571 30945 44508 
