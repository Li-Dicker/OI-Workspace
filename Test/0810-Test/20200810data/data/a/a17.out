Case #1: 13591099
Case #2: 15227741
Case #3: 14563662
Case #4: 13161368
Case #5: 13351212
Case #6: 14879966
Case #7: 14571332
Case #8: 14341894
Case #9: 14249396
Case #10: 13710249
